lib/srv/discovery/common/database.go

```diff
@@ -1257,7 +1257,7 @@ func MetadataFromRedshiftServerlessVPCEndpoint(endpoint *rsstypes.EndpointAccess
 
 // ExtraElastiCacheLabels returns a list of extra labels for provided
 // ElastiCache cluster.
-func ExtraElastiCacheLabels(cluster *elasticache.ReplicationGroup, tags []*elasticache.Tag, allNodes []*elasticache.CacheCluster, allSubnetGroups []*elasticache.CacheSubnetGroup) map[string]string {
+func ExtraElastiCacheLabels(cluster *ectypes.ReplicationGroup, tags []ectypes.Tag, allNodes []ectypes.CacheCluster, allSubnetGroups []ectypes.CacheSubnetGroup) map[string]string {
```

Comment:
FYI not in the scope of the SDK migration but we need to add cluster.Engine as the engine tag:
https://github.com/gravitational/teleport/issues/47507

> // The engine used in a replication group. The options are redis, memcached or
> // valkey.
> Engine *[string](https://pkg.go.dev/builtin#string)

Reply:
thanks!